daemon/algod/api/algod.oas2.json

```diff
@@ -2371,6 +2374,33 @@
           }
         }
       }
+    },
+    "/v2/experimental": {
```

Comment:
Without this addition, there would be no experimental endpoints. In theory this is fine, but it causes the generated file `daemon/algod/api/server/v2/generated/experimental/routes.go` to have compiler errors since an import is never used.

So I thought a reasonable way to solve this is to introduce a new experimental endpoint that's essentially a no-op.